apps/site/src/app/_components/Listbox/ListboxOptionGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ElementType } from 'react'
+
+type ListboxOptionGroupProps = {
+  label: string
+  as?: ElementType
+  children: React.ReactNode
+}
+
+export function ListboxOptionGroup({
```

Comment:
Would `ListboxGroupHeader` describe this component better?

```suggestion
export function ListboxGroupHeader({
```